src/build123d/joints.py

```diff
@@ -677,8 +680,8 @@ def symbol(self) -> Compound:
     def __init__(
         self,
         label: str,
-        to_part: Union[Solid, Compound] = None,
-        joint_location: Location = Location(),
+        to_part: Union[Solid, Compound, None] = None,
```

Comment:
Although I'm not a big fan of Optional as it seems redundant, I believe this is the correct way to do this:
```py
    to_part: Optional[Union[Solid, Compound]] = None,
    joint_location: Optional[Location] = None,
```